server/chat-plugins/the-studio.ts

```diff
@@ -303,7 +313,10 @@ class RecommendationsInterface {
 	}
 
 	async render(rec: Recommendation, suggested = false) {
-		const videoInfo = await YouTube.getVideoData(rec.url);
+		let videoInfo = null;
+		try {
+			videoInfo = await YouTube.getVideoData(rec.url);
+		} catch (e) {}
```

Comment:
I would prefer this gives an actual error reply

Reply:
Ok - you previously didn't do that so i assumed it wasn't necessary. One sec.